appwrite/sdk-for-go
# Auto close PR test

## What does this PR do?
This PR adds a GitHub workflow to automatically close pull requests and add a comment explaining that the repository is auto-generated and does not accept direct contributions. This is being tested on an unused SDK repository.

## Test Plan
To verify this change:
1. Create a new pull request in the unused SDK repository.
2. Observe that the workflow is triggered automatically.
3. Verify that a comment is added to the pull request with the specified text.
4. Confirm that the pull request is automatically closed.
5. Test reopening the pull request to ensure the workflow triggers again.

## Related PRs and Issues
N/A - This is a standalone change for repository management.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yea
